README.md

```diff
@@ -186,9 +186,9 @@ class AllAtOnceSpec extends FlatSpec with Matchers with BeforeAndAfterAll with G
 
   implicit val pc = PatienceConfig(Span(20, Seconds), Span(1, Second))
 
-  "all containers" should "be ready at the same time" in {
+  "all containers" should "be ready" in {
```

Comment:
Changed the test string, since the test isn't checking the containers should be ready "at the same time" (i.e. syncronized), but simply that eventually they all should be up. Right?